src/components/BulkEditList/BulkEditListResult/BulkEditInAppPreviewModal/BulkEditInAppPreviewModal.js

```diff
@@ -53,6 +56,7 @@ const BulkEditInAppPreviewModal = ({
   const { contentUpdate } = useContentUpdate({ id: bulkOperationId });
   const { bulkOperationStart } = useBulkOperationStart();
```

Comment:
please use isLoading here, it available for useBulkOperationStart

Reply:
answered before